src/modules/add_todo.js

```diff
@@ -0,0 +1,28 @@
+import TaskManager from './crud.js';
+
+const addTaskManager = () => {
+  const getTodos = JSON.parse(localStorage.getItem('todos'));
+  let getTaskList = [];
+  if (getTodos !== null) {
+    getTaskList = getTodos;
+  }
+
+  const task = document.querySelector('.todo-name');
+  const addbtn = document.querySelector('.add-to-list');
+
+  // create todo when task is added..
+  addbtn.addEventListener('click', () => {
+    if (task.value) {
+      const noteIndex = getTaskList.length;
+
+      const newTask = new TaskManager(task.value, noteIndex);
+      getTaskList.push(newTask);
+
+      // send todoList to localStorage
+      newTask.addTask(getTaskList);
+      task.value = '';
+    }
+  });
+};
+
```

Comment:
- [ ] as you can see in the picture below the index start on 0 but it should be start from 1 as required so please try do that in order to complete the requirements 
- Screenshot of the requirements
<img width="1199" alt="Screenshot 2022-12-08 at 13 26 12" src="https://user-images.githubusercontent.com/61790587/206446263-f0dc39b4-2ccc-40ed-b387-df8a01f76dc2.png">
<img width="1440" alt="Screenshot 2022-12-08 at 13 25 13" src="https://user-images.githubusercontent.com/61790587/206446273-82bcd49c-44be-4c8c-b2df-c64276f7ec8b.png">